supervisor/api/utils.py

```diff
@@ -25,22 +22,20 @@
 from ..utils import check_exception_chain, get_message_from_exception_chain
 from ..utils.json import JSONEncoder
 from ..utils.log_format import format_message
+from .const import CONTENT_TYPE_BINARY, HEADER_TOKEN, HEADER_TOKEN_OLD
 
 
 def excract_supervisor_token(request: web.Request) -> Optional[str]:
     """Extract Supervisor token from request."""
-    supervisor_token = request.headers.get(HEADER_TOKEN)
-    if supervisor_token:
+    if supervisor_token := request.headers.get(HEADER_TOKEN):
         return supervisor_token
 
-    # Remove with old Supervisor fallback
-    supervisor_token = request.headers.get(HEADER_TOKEN_OLD)
-    if supervisor_token:
+    # Old Supervisor fallback
```

Comment:
Should this have a "remove on" version?

Reply:
I guess we have to wait 2-3 years for that